### PR TITLE
docs: upgrade guide to v1.0.0

### PR DESCRIPTION
**Pull request by 0xHansLee**
# Description

Added instructions for upgrade to Kroma `v1.0.0`.

### CODE REVIEW

**Review by seolaoh — APPROVED** [review]
LGTM

File: upgrades/v1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+### Stop Kroma
+
+For upgrade, stop your Kroma full node or validator.
+```bash
+# for full node
+docker compose --profile fullnode down -v
+
+# for validator
+docker compose --profile validator down -v
```
Comment (ccrayz) [inline upgrades/v1.0.0.md]: I think removing the `-v` option will prevent users from having their volumes removed.

**Review by wo-o — APPROVED** [review]
LGTM

File: upgrades/v1.0.0.md (hunk 1 above)
Comment (0xHansLee) [reply upgrades/v1.0.0.md]: I see. I'll remove it.

**Review by ccrayz — APPROVED** [review]
LGTM

**Pull request merged**